packages/shopware-6-client/src/services/cartService.ts

```diff
@@ -0,0 +1,74 @@
+import { Cart } from "../interfaces/models/checkout/cart/Cart";
+import {
+  getCheckoutCartEndpoint,
+  getCheckoutCartProductEndpoint,
+  getCheckoutPromotionCodeEndpoint,
+  getCheckoutCartLineItemEndpoint
+} from "../endpoints";
+import { apiService } from "../apiService";
+import { ContextTokenResponse } from "../interfaces/response/ContextTokenResponse";
+
+export async function createCart(): Promise<ContextTokenResponse> {
+  const resp = await apiService.post(getCheckoutCartEndpoint());
+  let contextToken = resp.data["sw-context-token"];
+  return { contextToken };
+}
+
+export async function getCart(): Promise<Cart> {
+  const resp = await apiService.get(getCheckoutCartEndpoint());
+
+  return resp.data;
+}
+
+export async function addProduct(
+  productId: string,
+  newQuantity: number
```

Comment:
why is it called `newQuantity`? I believe it's just quantity :)
```suggestion
  quantity: number
```